docs/maps/urban.md

```diff
@@ -0,0 +1,40 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Urban
+
+***
+
+#### ![urbanicon](../assets/maps/urban/urban-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A fort built in a metropolis
```

Comment:
Could this be described without using a singular "fort"? Maybe something like: "A city that has become a battlefield"?